tests/setup_test.go

```diff
@@ -7008,3 +7008,88 @@ func TestProjectCommitStatusRedelivery(t *testing.T) {
 		}
 	})
 }
+
+func TestAgolaRunCounterVariableIsDefined(t *testing.T) {
```

Comment:
This test should be generalized tho check all the required environment variables and the name should be something like `TestRunRequiredEnvVariables`.